docs/user_guide.md

```diff
@@ -1326,6 +1331,97 @@ common arguments:
                         Set verbosity level.
 ~~~~~~~~~~~~~~~~~~~~~
 
+### <a name="source-components"></a> Source components (`components`)
+
+~~~~~~~~~~~~~~~~~~~~~
+usage: CodeChecker cmd components [-h] [--url PRODUCT_URL]
```

Comment:
What do you think to use the singular form `component` instead of components?
You add and delete them one by one.

Reply:
We use plural forms for `runs`, `results`, `products`. I do not know which would be better.